.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1 @@
+**Before submitting a pull request,** please make sure the following is done:
```

Comment:
Whats this Template supposed to do?

Reply:
@vichu1988 this file explain the pull request process . for more information please look below 
https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/